### PR TITLE
Map01

### PR DESCRIPTION
**Pull request by Alina-PANG**
1. Added in google map basic functions
2. code cleaning and styling
3. added in detecting current location function

### CODE REVIEW

File: src/main/webapp/css/main.css [hunk 1]
```diff
@@ -14,36 +14,53 @@
  * limitations under the License.
  */
 
+
 /* Prevent the page from becoming too wide. */
+
 body {
-  max-width: 100%;
+    max-width: 100%;
```
Comment (kiatchuan) [inline src/main/webapp/css/main.css]: Could we use two spaces, no tabs? This is based on the Airbnb ESLint config, for both CSS and JS files.

File: src/main/webapp/map.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
+<!--
```
Comment (kiatchuan) [inline src/main/webapp/map.html]: Probably don't need this copyright :)

File: src/main/webapp/css/main.css (hunk 1 above)
Comment (Alina-PANG) [reply src/main/webapp/css/main.css]: I am using a auto indentation styling extension in visual studio called JS-CSS-HTML Formatter, maybe that is why some of the indentation of the files have changed..

Comment (Alina-PANG) [reply src/main/webapp/css/main.css]: ok i just found another extension with eslint integration called prettier. Thanks for pointing out this issue!

File: src/main/webapp/map.html (hunk 2 above)
Comment (Alina-PANG) [reply src/main/webapp/map.html]: yep removed :)

**Comment by kiatchuan** [comment]
Please resolve conflicts, and ideally keep 2 spaces for HTML :)

**Comment by LiMengyang990726** [comment]
> Please resolve conflicts, and ideally keep 2 spaces for HTML :)

Conflicts resolved.

**Comment by LiMengyang990726** [comment]
Hi Hangzhi I think the HTML files are still in 4-space indentation, please change them to 2-space.

**Comment by Alina-PANG** [comment]
I just modified the css/html and js style, all the indentations are 2 spaces now. And there is no Eslint syntax highlights anymore in VSCode.

**Pull request merged**